packages/next/src/server/web/spec-extension/adapters/request-cookies.ts

```diff
@@ -144,29 +152,83 @@ export class MutableRequestCookiesAdapter {
               )
               try {
                 target.delete(...args)
+                return wrappedCookies
```

Comment:
fixes a small small bug -- the type signature of ResponseCookies says that `delete` should return `this` for chaining: `cookieStore.set(...).set(...)`, and we weren't doing that